content/style_reuse.md

```diff
@@ -261,7 +298,7 @@ and `panel-id`/`panel-link` values must be unique on the page.
     {{</* /foundation_tabs_panels */>}}
 
 
-## Fontawesome
+## Fontawesome Shortcode
 
 The Fontawesome shortcode will display any free [Font Awesome icon](https://fontawesome.com/icons) in a page.
```

Comment:
How do we resize or style Fontawesome icons?

Reply:
That comes after line 303.